helm-chart/templates/deployment.yaml

```
@@ -0,0 +1,34 @@
+              containerPort: 3000
+              protocol: TCP
+          envFrom:
+            - secretRef:
+                name: {{ include "formbricks.fullname" . }}-secrets
+          env:
+            {{- range $key, $value := .Values.env }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)